aten/src/ATen/native/native_functions.yaml

```diff
@@ -8814,6 +8814,19 @@
 #
 # See linalg_det as an example.
 
+- func: linalg_cholesky(Tensor self) -> Tensor
+  python_module: linalg
+  use_c10_dispatcher: full
+  variants: function
+  dispatch:
```

Comment:
Can we get rid of the dispatch keys here and in the out variant?

Reply:
It seems that with recent changes this is now a required thing to do.
https://github.com/pytorch/pytorch/tree/master/aten/src/ATen/native#choosing-the-right-dispatch-keyword
> Yes, but you still want to provide a numerically stable gradient formula instead of using autograd, write
dispatch:
  DefaultBackend: kernel

Otherwise, it hits [these line](https://github.com/pytorch/pytorch/blob/master/tools/autograd/gen_variable_type.py#L723-L728). Maybe I'm using it wrong.

Reply:
My mistake. I think you're right and this recently changed.